src/Elastic.Apm.GraphQL.HotChocolate/HotChocolateDiagnosticListener.cs

```diff
@@ -29,8 +31,10 @@ internal class HotChocolateDiagnosticListener : DiagnosticEventListener
 
         public override IActivityScope ExecuteRequest(IRequestContext context)
         {
-            ITransaction transaction = _apmAgent.Tracer.CurrentTransaction;
-            return new RequestActivityScope(context, transaction, _apmAgent);
+            ITransaction? transaction = _apmAgent.Tracer.CurrentTransaction;
+            return transaction != null
+                ? new RequestActivityScope(context, transaction, _apmAgent)
+                : _emptyActivityScope;
```

Comment:
use EmptyScope property from Base class

Reply:
ops, didn't see that there is an option to use it. Fixed